framework/src/media/MediaPlayerImpl.cpp

```diff
@@ -378,15 +378,15 @@ int MediaPlayerImpl::getVolume()
 	return ret;
 }
 
-void MediaPlayerImpl::getVolumePlayer(int &ret)
+void MediaPlayerImpl::getVolumePlayer(uint8_t &ret)
```

Comment:
int is correct to get AUDIO_MANAGER_DEVICE_NOT_SUPPORT

Reply:
If you want get default value when you call getVolume, then we can keep this as uint8_t
There are 2 options for getvolume and 1 option for setvolume (Assume that if driver provide fixed level , like a cx20921's gain)
1. Getvolume
 - Return AUDIO_MANAGER_DEVICE_NOT_SUPPORT 
 - return Default value of Device
2. SetVolume
 - Return AUDIO_MANAGER_DEVICE_NOT_SUPPORT

Reply:
Now the default value of the audio device is returned though the exclude config is set.